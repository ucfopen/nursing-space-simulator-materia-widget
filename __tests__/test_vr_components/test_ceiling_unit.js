import React from "react";
import renderer from "react-test-renderer";
import { Entity } from "aframe-react";
import Aframe from "aframe";

import CeilingUnit from "../../src/js/components/assets/ceiling_unit";

describe("CeilingUnit tests", () => {
	it("renders CeilingUnit correctly", () => {
		const mockClick = jest.fn();

		const ceilingUnitTree = renderer
			.create(<CeilingUnit x={1} y={1} />)
			.toJSON();

		expect(ceilingUnitTree).toMatchSnapshot();
	});
});
