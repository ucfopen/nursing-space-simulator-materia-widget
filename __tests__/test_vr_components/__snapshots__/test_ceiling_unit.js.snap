// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CeilingUnit tests renders CeilingUnit correctly 1`] = `<a-plane />`;
